src/main/resources/db/changelog/changes/v1.1.0/sql/entity-type-definitions/insert-languages-definition.sql

```diff
@@ -0,0 +1,18 @@
+INSERT INTO entity_type_definition (id, derived_table_name, definition)
+     VALUES ('a435d29f-ff11-4b7a-8a5f-3c5505452208','drv_languages', '{
+         "id": "a435d29f-ff11-4b7a-8a5f-3c5505452208",
+         "name": "drv_languages",
+         "fromClause": "drv_languages",
+         "root": true,
+         "private" : true,
+         "columns": [
+             {
+               "name": "languages",
```

Comment:
I've read the word "languages" so many times in this PR that it doesn't look like a real word anymore

Reply:
creating illusions lol :)